docs/Configuration/yaml-files.md

```diff
@@ -318,11 +318,20 @@ The `software` section allows you to configure packages and Apple App Store apps
 software:
   packages:
     - path: ../lib/software-name.package.yml
+    - path: ../lib/software-name2.package.yml
+      labels_include_any:
+        - Engineering
+        - Customer Support
   # path is relative to default.yml, teams/team-name.yml, or teams/no-team.yml
   app_store_apps:
     - app_store_id: '1091189122'
+    - app_store_id: '1477376905'
+      labels_include_any:
+        - Engineering
 ```
 
+Use `labels_include_any` to make software available for install or installed by policy automation only on hosts that have all those labels. Use `labels_exclude_any` to make software available for install or installed by policy automation only on hosts that don't have any of those labels.
```

Comment:
These sentences are starting to get really complex; looking for a way to make this information easier to parse. Maybe:
```suggestion
When making software available for install or installable by policy automation, you can include one of the following arrays of labels to narrow the scope of software installs:
- `labels_include_any`: matches hosts that have all labels in the list
- `labels_exclude_any`: matches hosts that don't have any labels in the list
```
(cc @noahtalerman in case you have any ideas/feedback about how to word this)

Reply:
I think let's focus on the feature: targeting.

I think what we have for configuration profiles [here](https://fleetdm.com/docs/configuration/yaml-files#macos-settings-and-windows-settings) does this pretty well:
![Screenshot 2024-12-10 at 4 42 28 PM](https://github.com/user-attachments/assets/491d368d-b52c-42d8-8693-7d983263cae7)

FYI @rachaelshaw and @marko-lisica I made changes to use the "targeting" language.